src/Control/Monad/Freer/Interpretation.hs

```diff
@@ -0,0 +1,211 @@
+{-# LANGUAGE CPP                   #-}
+{-# LANGUAGE DataKinds             #-}
+{-# LANGUAGE MonoLocalBinds        #-}
+#if __GLASGOW_HASKELL__ >= 806
+{-# LANGUAGE QuantifiedConstraints #-}
+#endif
+{-# LANGUAGE RankNTypes            #-}
+{-# LANGUAGE TypeOperators         #-}
```

Comment:
A bunch of these are already on by default, so please remove the ones that aren’t strictly necessary.